Sources/Testing/Traits/Trait.swift

```diff
@@ -70,45 +132,14 @@ extension Trait {
   public var comments: [Comment] {
     []
   }
+
+  public var customTestExecutor: CustomTestExecutor? {
```

Comment:
We don't want the default implementation to return `nil` if a type _other than `Never` is used. We should limit it to:

```swift
extension Trait where CustomTestExecutor == Never {
  ...
}
```